src/index.js

```diff
@@ -48,427 +48,250 @@ const removeEventListener = (event) => {
   }
 };
 
-const logUninitializedAccessError = (callingMethod) => {
-  console.warn("ERROR: Failed to execute " + callingMethod + "() - please ensure the AppLovin MAX React Native module has been initialized by calling 'AppLovinMAX.initialize(...);'!");
+const runIfInit = (callingMethodName, callingMethod, ...params) => {
```

Comment:
I would name it more descriptively - `runIfInitialized` instead, since `Init` could be read as a noun as opposed to adjective (for which it would've been `init'd`.